WorkflowExample/WorkflowExample/Ordering/ReviewOrderViewController.swift

```diff
@@ -11,38 +11,39 @@ import Workflow
 import WorkflowUIKit
 import UIKit
 
-class ReviewOrderViewController: UIWorkflowItem<Order, Order?>, StoryboardLoadable {
-    var order: Order?
+class ReviewOrderViewController: UIWorkflowItem<Order, Order>, StoryboardLoadable {
```

Comment:
As discussed, we'll want to have added documentation that `StoryboardLoadable` should be attached to the class instead of an extension to get the best experience (at this time).

Reply:
As well as `FlowRepresentable` for those who want to programmatically create their view controllers.

Reply:
Added docs